letsencrypt/tests/auth_handler_test.py

```diff
@@ -148,6 +148,42 @@ def test_perform_failure(self):
         self.assertRaises(
             errors.AuthorizationError, self.handler.get_authorizations, ["0"])
 
+    @mock.patch("acme.challenges.HTTP01Response.simple_verify")
+    @mock.patch("acme.challenges.TLSSNI01Response.simple_verify")
+    def test_simple_verify(self, mock_http01_verify, mock_tlssni01_verify):
+        from acme import jose
+        from letsencrypt.tests import test_util
```

Comment:
The rule is: if it's the module that your testing then import in the function (in this file only `from letsencrypt import auth_handler`), otherwise import at the top.